### DISCUSSION TITLE
SDK Custom Splash Image - Sizes to include? ? ?

### DISCUSSION THREAD

**Discussion by dmunsie** [airsdk/Adobe-Runtime-Support#3161] [General]
Using the guidelines here:
https://docs.airnativeextensions.com/docs/tutorials/android-splash-screen/

I'm able to include a splash screen but depending on the resolution of the device it scales too big or too small. The documentation states:

When using this method we advise that you add all the different dpi versions of your image to correctly support different screen sizes: Read more here: https://developer.android.com/guide/practices/screens_support.html

"we advise that you add all the different dpi versions of your image to correctly support different screen sizes"

I couldn't find anything to help me with this, what are the filenames, what sizes should they be? Thx for any help.

**Comment by dmunsie** [comment]
Even better, is there a way just to include 1 image and have it scale/stretch to the device? Thx.

**Reply by ajwfrost** [reply]
The Android documentation talks more about pixel density than about resolution, so it's not 100% clear from looking at it, but I think that's what you'd need to use.

There is a potential way to get it to scale, I think, which is to put your image in the "drawable-nodpi" folder.

But what they recommend is to create multiple resolutions of your image and put them into the appropriate folders (with the same filename):
* drawable-ldpi = image for low density screens (120dpi)
* drawable-mdpi = normal image for medium density screens (160dpi)
* drawable-hdpi = image for high density screens (240dpi)
* drawable-xhdpi = image for extra-high density screens (320dpi)
* drawable-xxhdpi = image for extra-extra-high density screens (480dpi)
* drawable-xxxhdpi =image for extra-extra-extra-high density screens (640dpi) 

Looks like they ran out of superlatives, can't wait for extra-extra-extra-extra high...

thanks

**Reply by 2jfw** [reply]
> Looks like they ran out of superlatives, can't wait for extra-extra-extra-extra high...

Yeah, similar to Triple-A games now also starting to become Quadruple-A games (see Ubisoft, Skull and Bones) 😂
